estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,68 @@
+from odoo import fields, models, api, exceptions
+
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Estate Property Offer"
+    _order = "price desc"
+
+    price = fields.Float(string="Price", required=True)
+    status = fields.Selection([
+        ("accepted", "Accepted"),
+        ("rejected", "Rejected"),
+    ], copy=False, string="Status")
+
+    partner_id = fields.Many2one(string="Buyer", comodel_name="res.partner", required=True)
+    property_id = fields.Many2one(string="Property", comodel_name="estate.property", required=True)
+    property_type_id = fields.Many2one(string="Property Type", comodel_name="estate.property.type", related="property_id.property_type_id", store=True)
```

Comment:
why do you need `store` here?

Reply:
It was required by chapter 12: stat button and the related column isn't stored by default.
![image](https://github.com/odoo/tutorials/assets/72211774/307b22e4-def2-436e-8982-724fea3849c2)